app/services/recurring_contribution/update_status.rb

```diff
@@ -13,6 +14,8 @@ def self.process(request, resource)
 
   def process
     update_status if request_source_is_pagarme?
+  rescue ArgumentError
```

Comment:
o problema é que to usando o `.update`, o erro é disparado pq ele já tenta salvar. Vou ver uma outra forma aqui